### PR TITLE
SONARJAVA-3493 Simplify serialized method behaviors

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

File: java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehaviorJsonAdapter.java [hunk 1]
```diff
@@ -36,20 +36,18 @@
...
+import org.sonar.java.se.constraint.Constraint;
 import org.sonar.java.se.constraint.ConstraintsByDomain;
 import org.sonar.java.se.constraint.ObjectConstraint;
 
 public class MethodBehaviorJsonAdapter implements JsonSerializer<MethodBehavior>, JsonDeserializer<MethodBehavior> {
 
-  private static final String JSON_CONSTRAINT_DELIMITER = ".";
-  private static final String JSON_IS_EXCEPTIONAL = "isExceptional";
   private static final String JSON_THROWN_EXCEPTION = "exception";
   private static final String JSON_RESULT_CONSTRAINTS = "resultConstaint";
```
Comment (quentin-jaquier-sonarsource) [inline java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehaviorJsonAdapter.java:46]: Missing an `r` here.

**Review by quentin-jaquier-sonarsource — APPROVED** [review]
We missed a small typo in the first PR, I believe it makes sense to fix it now.

File: java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehaviorJsonAdapter.java (hunk 1 above)
Comment (m-g-sonar) [reply java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehaviorJsonAdapter.java:46]: aaaaaaaaaghhhhhhhh (with a missing `r`)

**Review by quentin-jaquier-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**